src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -425,7 +425,9 @@ internal static FlowAnalysisAnnotations GetFlowAnalysisAnnotations(this Symbol?
             return symbol switch
             {
                 MethodSymbol method => method.ReturnTypeFlowAnalysisAnnotations,
-                PropertySymbol property => property.GetOwnOrInheritedGetMethod()?.ReturnTypeFlowAnalysisAnnotations ?? FlowAnalysisAnnotations.None,
+                PropertySymbol property =>
+                    (property.GetOwnOrInheritedGetMethod()?.ReturnTypeFlowAnalysisAnnotations ?? FlowAnalysisAnnotations.None)
```

Comment:
The pattern of `property.GetOwnOrInheritedGetMethod()?.ReturnTypeFlowAnalysisAnnotations ?? FlowAnalysisAnnotations.None` seems like it's repeated a bit in our code. It's here now as well as being inside of `SymbolExtensions.GetFlowAnalysisAnnotations`. Why do we not just have this as a property on `PropertySymbol` as we do for `MethodSymbol`? Feels wrong to have it repeated (avenue for future bugs if we only fix one instance of it)

Reply:
It does feel like the implementation in here should be consolidated with NullableWalker.GetLValueAnnotations.

Reply:
I implemented it as an extension method for expediency. It feels like properties are special because we usually only get their flow analysis annotations from the accessors, and only in the special case of assigning to a read-only auto-property in a constructor do we look at the property itself to learn its flow analysis annotations.